docs/getting_started/user-management.md

```diff
@@ -0,0 +1,135 @@
+This can be done when launching Rubrix from Python or with the provided `docker-compose.yml`. Read below for more details on the different options.
+
+## Default user
+
+By default, if you don't configure a `users.yml` file, your Rubrix instance is pre-configured with the following default user:
+
+- username: `rubrix`
+- password: `1234`
+- api_key: `rubrix.apikey`
+
+for security reasons we recommend changing at least the password and API key.
+
+### How to override the default API key
+
+To override the default API key you can set the following environment variable before launching the server:
+
+```bash
+export RUBRIX_LOCAL_AUTH_DEFAULT_APIKEY=new-apikey
+```
+
+
+### How to override the default user password
+
+To override the password, you must set an environment variable that contains an already hashed password.
+You can use `htpasswd` to generate a hashed password:
+
+```bash
+htpasswd -nbB "" my-new-password
+:$apr1$n0C4S20a$noG.3yWxH1OIKfFITgzl30
+```
+
+Afterwards, set the environment variable omitting the first `:` character (in our case `$apr1$n0C4...`):
+
+```bash
+export RUBRIX_LOCAL_AUTH_DEFAULT_PASSWORD="<generated_user_password>"
+```
+
+Alternatively, you can also generate the hash using passlib's CryptContext: E.g., by running the following in a python console:
+
+```python
+from passlib.context import CryptContext
+print(CryptContext(schemes=["bcrypt"], deprecated="auto").hash('password'))
+```
+
+## How to add new users and workspaces
+
+To configure your Rubrix instance for various users, you just need to create a yaml file as follows:
+```yaml
+#.users.yaml
+# Users are provided as a list
+- username: user1
+  hashed_password: <generated-hashed-password> # See the previous section above
+  api_key: "ThisIsTheUser1APIKEY"
+  workspaces: [] # This user will only have her user workspace available
+- username: user2
+  hashed_password: <generated-hashed-password> # See the previous section above
+  api_key: "ThisIsTheUser2APIKEY"
+  workspaces: ['client_projects'] # access to her user workspace and the client_projects workspace
+- username: user3
+  hashed_password: <generated-hashed-password> # See the previous section above
+  api_key: "ThisIsTheUser2APIKEY" # this user can access all workspaces (including
+- ...
+```
+
+Then point the following environment variable to this yaml file before launching the server:
+
+```bash
+export RUBRIX_LOCAL_AUTH_USERS_DB_FILE=/path/to/.users.yaml
+```
+
+If everything went well, the configured users can now log in and their annotations will be tracked with their usernames.
+
+
+### Using docker-compose
+
+Make sure you create the yaml file above in the same folder as your `docker-compose.yaml`. You can download the `docker-compose` from this [URL](https://git.io/rb-docker):
+
+Then open the provided ``docker-compose.yaml`` and configure your Rubrix instance as follows:
+
+{{ '```yaml\n{}\n```'.format(dockercomposeuseryaml) }}
```

Comment:
Could we include just the `dockercomposeyaml` var as is and prepend the `yaml\n` inside the variable?

Reply:
of courrrrse! I applied your suggestions